src/base/system_parameter.c

```diff
@@ -6773,6 +6792,12 @@ prm_load_by_section (INI_TABLE * ini, const char *section, bool ignore_section,
 	    }
 	}
 
+      if (strcmp (section, "common") == 0 && strcmp (prm->name, PRM_NAME_JAVA_STORED_PROCEDURE_PORT) == 0)
+	{
+	  /* ignore specifying port for java stored procedure in common section */
+	  continue;
+	}
+
```

Comment:
It seems that ignoring the common configuration of JSP is confusing to users because only one DB instance is usually created on DB engine.
What about raising an error rather than ignoring it?